src/schema/v2/schema.ts

```diff
@@ -160,6 +161,7 @@ const rootFields = {
   salesConnection: SalesConnectionField,
   searchConnection: Search,
   show: Show,
+  viewingRoom: ViewingRoom,
```

Comment:
Added viewRoom to be accessible as a node in the local schema

Reply:
Might need a pluralized loader to query for a list of viewing rooms? Looks like there is a pattern here for single lookups and list lookups - doing some research

Reply:
Ah another issue here, looks like the request needs to be authenticated to get viewing rooms data here. In this current set up, the gravity loader is in the public space

Reply:
Oooo interesting - @jpotts244 `viewingRoom` should actually be available as a top-level field already that can be queried with an ID, it's just stitched in from Gravity.

That said, I definitely don't fully understand how loading a VR object by ID in Metaphysics works. I can go open up a staging console and query and we're fine, but I'm less familiar with how this would work when Vortex is hitting Metaphysics.

Happy to chat about this when you're back, or maybe @ansor4 understands what's going on here better than I do?